### PR TITLE
changed sub_jwk to cnf claim for holder binding

### PR DESCRIPTION
**Pull request by Sakurann**
addresses Issue #77

### CODE REVIEW

File: main.md [hunk 1]
```diff
@@ -802,7 +802,7 @@ allows for the release of individual members of the address claim separately.
 ```json
 {
   "iss": "https://example.com/issuer",
-  "sub_jwk": {
+  "cnf": {
```
Comment (bc-pi) [inline main.md:805]: There's a layer of nesting missing here per https://datatracker.ietf.org/doc/html/rfc7800#section-3.2 that uses "jwk" as the confirmation method name.  i.e.:

```
"cnf":{
    "jwk":{
        "kty": "RSA",
          ...  n and e ... 
 }}```

File: main.md [hunk 2]
```diff
@@ -932,7 +932,7 @@ The following shows the resulting SD-JWT payload:
...
```
Comment (bc-pi) [inline main.md:935]: here too https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/119/files#r919304849

**Comment by peppelinux** [comment]
@Sakurann considering the last OOP refactor on the python code and the revision of Brian probably it would be easier to close this PR and starting a new one on top of the current master branch, the changes in the code and in the text are very trivial

**Comment by Sakurann** [comment]
@peppelinux ok, please do!

**Review by peppelinux — APPROVED** [review]

**Review by danielfett — APPROVED** [review]

**Pull request merged**